examples/cra-ts-essentials/src/stories/Button.babelproposal.deleteme.tsx

```diff
@@ -0,0 +1,101 @@
+// ESLINT AND TSIGNORE ADDED JUST TO NOT HAVE ERRORS WHILE READING THE PROPOSAL
```

Comment:
If we are to write a babel plugin, this is a proposal for the steps to do it. This file should be deleted later